staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -313,8 +313,24 @@ type ListOptions struct {
 	// event containing a ResourceVersion after which the server
 	// continues streaming events.
 	SendInitialEvents *bool
+
+	// AggregateErrFn, if provided, will be invoked so the error(s)
+	// incurred during the list operation can be aggregated.
+	// The default is nil, and and maintains backward compatibility, which is
+	// the list operation will abort and return the first error it encounters.
+	// - key: identifies the object in the underlying storage
+	// - err: the error that occurred while retrieving the given object from the storage
+	// - done: if true, the list operation will abort, otherwise, it will continue
+	AggregateErrFn func(key string, err error) (done bool)
```

Comment:
Can't we be super explicit what this does instead of allowing an arbitrary function? This method here is a bad abstraction anyway, let's not pretend it isn't.

Reply:
this abstraction has been removed